samples/Shared/UnitTestFramework/MetadataReferences.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Linq;
+using System.Reflection;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+
+#if !NETSTANDARD1_5
+using System;
+#endif
+
+namespace Roslyn.UnitTestFramework
+{
+    /// <summary>
+    /// Metadata references used to create test projects.
+    /// </summary>
+    public static class MetadataReferences
```

Comment:
[This](https://github.com/GuOrg/Gu.Roslyn.Asserts/blob/master/Gu.Roslyn.Asserts/MetadataReferences.cs#L23) is a pretty useful helper method that can fit in this class.

Reply:
I'll leave this as a follow-up item. I agree it seems useful.